### DISCUSSION TITLE
Problems with combo font alignment

### DISCUSSION THREAD

**Discussion by MUGEN-Slayer** [ikemen-engine/Ikemen-GO#935] [Feature Request] [answered]
Lifebars have a bug regarding combo font alignment, where the P1 team's combo font cannot be right aligned and the P2 team's font cannot be left aligned. While this is something present in MUGEN, I've never seen anyone make use of this "feature".
While I don't remember seeing commercial games use left alignment for the P2 team's combo font, it's a non-existent possibility anyway as it's not possible in IKEMEN. 
However, it is possible to see that several commercial games use right alignment for the P1 team combo. I'll share some examples, check the timestamps:
- Arcana Heart: https://youtu.be/ZgY0NM5rvMk?t=12
- Guilty Gear Xrd Rev 2: https://youtu.be/iGDvOr3x4Tk?t=14
- The King of Fighters 2002 UM: https://www.youtube.com/watch?v=nToxpky7rLA
- The King of Fighters XV: https://youtu.be/YGaSmwVkaMo?t=12
- Samurai Shodown IV: https://www.youtube.com/watch?v=Nl31Aj82cbw
- Samurai Shodown 2019: https://youtu.be/Uf9f3Vb2Yos?t=187
- Street Fighter Alpha 1: https://youtu.be/3BasIHrlbO4?t=24
- Street Fighter Alpha 2: https://www.youtube.com/watch?v=3ig-E8sBK18 
- Street Fighter V: https://www.youtube.com/watch?v=MfH3lLvsAOs

And there are more examples, but above is enough to show that different games from different companies and eras have this quirk regarding how the combo font is displayed. So it would be great to be able to emulate that, with alignments working correctly.

Because now it's like this:

https://user-images.githubusercontent.com/68881657/205217274-046d52e3-43fb-42be-924e-00aec11ff7ed.mp4

https://user-images.githubusercontent.com/68881657/205217291-63a029dd-c2a0-437b-84b9-1167087e9b87.mp4

[fight.zip](https://github.com/ikemen-engine/Ikemen-GO/files/10138174/fight.zip)

**Comment by K4thos** [comment, marked as answer]
if it works like in mugen than the way how it works currently is intentional.
